packages/store/types/tsconfig.json

```diff
@@ -19,7 +19,7 @@
     "typeRoots": ["node_modules/@types"],
     "baseUrl": ".",
     "paths": {
-      "@angular/*": ["../../../node_modules/@angular/*"],
+      "@angular/core": ["../../../node_modules/@angular/core/core.d.ts"],
```

Comment:
Interesting, what is the reason for this change?

Reply:
It turns out that in reality the NGXS library uses only @angular/core in source code (use injectable, ngModule). Therefore, there is no need to specify any ng libraries. The most surprising surprise was that Angular now compiles its library files differently. They used to generate index.d.ts in the output. Therefore, everything worked for us. Now core.d.ts is generated (if it @angular/core), and for example testing.d.ts (if it @angular/core/testing).